### PR TITLE
WIP: do not merge: testing hcp custom cnfqe1 deployment

### PR DESCRIPTION
**Pull request by rshemtov13**

### CODE REVIEW

**Comment by rshemtov13** [comment]
/pj-rehearse periodic-ci-openshift-release-master-nightly-4.17-e2e-telco5g-hcp-cnftests

**Comment by rshemtov13** [comment]
/pj-rehearse periodic-ci-openshift-release-master-nightly-4.17-e2e-telco5g-hcp-cnftests

**Comment by rshemtov13** [comment]
/pj-rehearse periodic-ci-openshift-release-master-nightly-4.17-e2e-telco5g-hcp-cnftests

File: ci-operator/step-registry/telco5g/hcp-cluster-setup/telco5g-hcp-cluster-setup-commands.sh [hunk 1]
```diff
@@ -104,7 +108,7 @@ cat << EOF > $SHARED_DIR/release-cluster.yml
   tasks:
 
   - name: Release cluster from job
-    command: python3 ~/telco5g-lab-deployment/scripts/upstream_cluster_all.py --release-cluster $CLUSTER_NAME
+    command: python3 ~/telco5g-lab-deployment/scripts/upstream_cluster_all.py --release-cluster $CLUSTER_NAME -s
```
Comment (sshnaidm) [inline ci-operator/step-registry/telco5g/hcp-cluster-setup/telco5g-hcp-cluster-setup-commands.sh:111]: comment this playbook to leave the environment

Comment (rshemtov13) [reply ci-operator/step-registry/telco5g/hcp-cluster-setup/telco5g-hcp-cluster-setup-commands.sh:111]: ok got it

File: ci-operator/step-registry/telco5g/hcp-cluster-setup/telco5g-hcp-cluster-setup-commands.sh [hunk 2]
```diff
@@ -63,13 +63,17 @@ ping ${BASTION_IP} -c 10 || true
...
 ADDITIONAL_ARG="-e $CL_SEARCH --topology 1b1v --topology sno"
 
 cat << EOF > $SHARED_DIR/get-cluster-name.yml
 ---
 - name: Grab and run kcli to install openshift cluster
   hosts: bastion
   gather_facts: false
+  vars:
+    cluster:
+      {"cnfqe1": {"port": 6443, "ip", "10.46.100.20", "hvip": 10.46.100.1}}
```
Comment (sshnaidm) [inline ci-operator/step-registry/telco5g/hcp-cluster-setup/telco5g-hcp-cluster-setup-commands.sh:76]: should be `"ip": "10.46.100.20"`

Comment (rshemtov13) [reply ci-operator/step-registry/telco5g/hcp-cluster-setup/telco5g-hcp-cluster-setup-commands.sh]: okay understood changed

**Comment by rshemtov13** [comment]
/pj-rehearse periodic-ci-openshift-release-master-nightly-4.17-e2e-telco5g-hcp-cnftests

**Comment by sshnaidm** [comment]
/pj-rehearse periodic-ci-openshift-release-master-nightly-4.17-e2e-telco5g-hcp-cnftests

**Comment by rshemtov13** [comment]
/pj-rehearse periodic-ci-openshift-release-master-nightly-4.17-e2e-telco5g-hcp-cnftests

**Comment by rshemtov13** [comment]
/pj-rehearse periodic-ci-openshift-release-master-nightly-4.17-e2e-telco5g-hcp-cnftests

**Comment by rshemtov13** [comment]
/pj-rehearse periodic-ci-openshift-release-master-nightly-4.17-e2e-telco5g-hcp-cnftests

**Comment by rshemtov13** [comment]
/pj-rehearse periodic-ci-openshift-release-master-nightly-4.17-e2e-telco5g-hcp-cnftests

**Comment by rshemtov13** [comment]
/pj-rehearse periodic-ci-openshift-release-master-nightly-4.17-e2e-telco5g-hcp-cnftests

**Comment by rshemtov13** [comment]
/pj-rehearse periodic-ci-openshift-release-master-stable-4.17-e2e-telco5g-hcp-cnftests

**Comment by rshemtov13** [comment]
/pj-rehearse periodic-ci-openshift-release-master-nightly-4.17-e2e-telco5g-hcp-cnftests

**Comment by rshemtov13** [comment]
/pj-rehearse periodic-ci-openshift-release-master-nightly-4.17-e2e-telco5g-hcp-cnftests

**Comment by rshemtov13** [comment]
/pj-rehearse periodic-ci-openshift-release-master-nightly-4.17-e2e-telco5g-hcp-cnftests

**Comment by rshemtov13** [comment]
/pj-rehearse periodic-ci-openshift-release-master-nightly-4.17-e2e-telco5g-hcp-cnftests

**Comment by rshemtov13** [comment]
/pj-rehearse periodic-ci-openshift-release-master-nightly-4.17-e2e-telco5g-hcp-cnftests

**Comment by rshemtov13** [comment]
/pj-rehearse periodic-ci-openshift-release-master-nightly-4.17-e2e-telco5g-hcp-cnftests

**Comment by sshnaidm** [comment]
/pj-rehearse periodic-ci-openshift-release-master-nightly-4.17-e2e-telco5g-hcp-cnftests

**Comment by rshemtov13** [comment]
/pj-rehearse periodic-ci-openshift-release-master-nightly-4.17-e2e-telco5g-hcp-cnftests

**Pull request closed without merging**